src/components/Login/LoginMobileNumberCard.tsx

```diff
@@ -79,6 +80,12 @@ export const LoginMobileNumberCard: FunctionComponent<LoginMobileNumberCard> = (
   const [countryCode, setCountryCode] = useState("+65");
   const [mobileNumberValue, setMobileNumberValue] = useState("");
 
+  useEffect(() => {
```

Comment:
was thinking if you want to do a hook so that this chunk dont need to copy paste everywhere haha

Reply:
Yea was thinking if it's possible, for the on mount navigation events it'll work, for the other kinds such as when `loginStage` changes, not sure if it's beneficial since I'll essentially be replicating the useEffect logic with the dependency array. Any idea how this can be done?